src/panels/lovelace/cards/hui-sensor-graph-card.js

```diff
@@ -0,0 +1,273 @@
+import { LitElement, html } from '@polymer/lit-element';
+
+import '../../../components/ha-card.js';
+import '../../../components/ha-icon.js';
+
+import computeStateName from '../../../common/entity/compute_state_name.js';
+import stateIcon from '../../../common/entity/state_icon.js';
+
+import EventsMixin from '../../../mixins/events-mixin.js';
+
+class HuiSensorGraphCard extends EventsMixin(LitElement) {
+  set hass(hass) {
+    this._hass = hass;
+    const entity = hass.states[this.config.entity];
+    if (entity && this.entity !== entity) {
+      this.entity = entity;
+      this._getHistory();
+    }
+  }
+
+  static get properties() {
+    return {
+      _hass: Object,
+      config: Object,
+      entity: Object,
+      line: String
+    };
+  }
+
+  setConfig(config) {
+    if (!config.entity || config.entity.split('.')[0] !== 'sensor') {
+      throw new Error('Specify an entity from within the sensor domain.');
+    }
+
+    config.icon = config.icon || false;
+    config.hours_to_show = config.hours_to_show || 24;
+    config.accuracy = Number(config.accuracy) || 10;
+    config.height = Number(config.height) || 100;
+    config.line_color = config.line_color || 'var(--accent-color)';
+    config.line_width = Number(config.line_width) || 5;
+
+    this.config = config;
+  }
+
+  shouldUpdate(changedProps) {
+    const change = (
+      changedProps.has('entity')
+      || changedProps.has('line')
+    );
+    return change;
+  }
+
+  render({ config, entity, line } = this) {
+    return html`
+      ${this._style()}
+      <ha-card ?group=${config.group} @click='${e => this._handleClick(e)}'
+        ?more-info=${config.more_info}>
+        <div class='flex'>
+          <div class='icon'>
+            <ha-icon icon='${this._computeIcon(entity)}'></ha-icon>
+          </div>
+          <div class='header'>
+            <span class='name'>${this._computeName(entity)}</span>
+          </div>
+        </div>
+        <div class='flex info'>
+          <span id='value'>${entity.state}</span>
+          <span id='measurement'>${this._computeUom(entity)}</span>
+        </div>
+        <div class='graph'>
+          <div>
+            ${line ? html`
```

Comment:
You need to import `svg` from `lit-element` and use that to render the svg instead of `html`.